test/gtest/uct/test_error_handling.cc

```diff
@@ -27,6 +27,10 @@ class test_error_handling : public uct_test {
             GetParam()->tl_name == "dc" || GetParam()->tl_name == "dc_mlx5") {
             set_config("RC_TIMEOUT=0.0001"); /* 100 us should be enough */
             set_config("RC_RETRY_COUNT=2");
+        } else if (GetParam()->tl_name == "ud") {
+            set_config("TIMEOUT=1s");
+        } else if (GetParam()->tl_name == "ud_mlx5") {
+            set_config("UD_TIMEOUT=1s");
```

Comment:
[ RUN      ] ud/test_error_handling.peer_failure/0
../../../test/gtest/common/test.cc:66: Failure
Failed
Invalid UCS configuration for UD_TIMEOUT : 1s, error message: No such element(-12)